dist/index.html

```diff
@@ -3,17 +3,86 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- font awesome file -->
+    <link rel="stylesheet" href="./assets/css/font-awesome.min.css" />
+
     <link rel="stylesheet" href="./assets/css/main.css" />
     <title>Charity</title>
   </head>
   <body>
     <header>
-      <section class="contact">
-        Welcome text and social media Links
+      <section class="contact bg-c100 text-c700 py-2">
+        <div class="container flex justify-between">
+          <div class="welcome-text text-sm">
+            Welcome to the best
+            <span class="text-c300 underline italic">Lovims</span> charity
+            platform
+          </div>
+          <div class="social flex text-sm">
+            <div>Follow us:</div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-twitter" aria-hidden="true"></i>
+              </a>
+            </div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-facebook-square" aria-hidden="true"></i>
+              </a>
+            </div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-pinterest-p" aria-hidden="true"></i>
+              </a>
+            </div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-instagram" aria-hidden="true"></i>
+              </a>
+            </div>
+          </div>
+        </div>
       </section>
 
-      <section>
-        Logo And Contact Info
+      <section class="logo-section py-8">
+        <div class="container py-2 flex">
+          <div class="logo w-1/2">
+            <img src="assets/Logo@1X.png" alt="" />
+          </div>
+
+          <div class="contact-info flex justify-between w-full">
+            <div class="email flex border-r border-c700 pr-8">
```

Comment:
You did a great job of choosing flexbox to style this part. 
You just need to align the content of child elements vertically. 
Ex: _**This will align items vertically **_
> ` .parent {
>     display: flex;
>     align-items: center;
> } `
You can check this link for complete guide to flex box [Flex box Guide](https://cssreference.io/flexbox/#align-content).
And this is a good article also [Article about flex-box ](https://www.smashingmagazine.com/2018/08/flexbox-alignment/)
![alignment](https://user-images.githubusercontent.com/8650435/79673532-8bbdf080-81eb-11ea-8101-e32628914121.PNG)

Reply:
It is perfectly aligned. 👍